### ISSUE TITLE
Fix aria-label value for localized Dialog

### ISSUE THREAD

**Issue by Iankodj** [telerik/kendo-ui-core#2282]
### Bug report

Localization resources append text to the close button, but aria-label stays with value "Close".
### Reproduction of the problem

http://runner.telerik.io/fullscreen/Icuno
### Expected/desired behavior

WAI-ARIA:
- [x] aria-label should be assigned dynamically to the messages.close property and eventually localized;

Localization:  
- [x] messages.close should not add text to the close button of the dialog
### Environment
- **Kendo UI version:** 2016.3.914
- **jQuery version:** 1.12.4
- **Browser:** [all]
